### ISSUE TITLE
Install path for submodules annoys homebrew

### ISSUE THREAD

**Issue by famousj** [urbit/urbit#1005]
I was having problems with homebrew, so I ran `brew doctor` and I got these warnings:

```
Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libed25519.0.dylib
  /usr/local/lib/libh2o.0.dylib
  /usr/local/lib/liblibscrypt.0.dylib
  /usr/local/lib/liblibuv.1.dylib
  /usr/local/lib/libmurmur3.0.dylib
  /usr/local/lib/libsoftfloat3.3.dylib
```

Ditto for headers and static libraries.

I'm assuming these are urbit submodules.  Any chance those could get installed somewhere else, or would that cause different problems?

**Comment by galenwp** [comment]
Just to be clear — you were having unrelated-to-urbit problems with homebrew before getting this message?

**Comment by famousj** [comment]
Yes.  I was running `brew doctor` in the first place to fix something.

This is not, in my opinion, an error.  But `homebrew` considers it an error to have unexpected libraries in `/usr/local`.  I can see this causing problems for people who don't usually watch the screen while `make` is running.

**Comment by keatondunsford** [comment]
Seems like we could have them installed to a hidden `~/.urbit/` directory and have the binary read from that location. This would make it easy for people to upgrade or delete the submodules that don't conform to the Homebrew way of installing and upgrading. Many projects install dependency, configuration, and data files in this way.

**Comment by joemfb** [comment]
Fixed in #1093. For anyone who ended up with those files from earlier urbit installations, they can be removed without issue.
